lib/PEXv2.ts

```diff
@@ -198,7 +201,7 @@ export class PEXv2 {
    */
   public async verifiablePresentationFromAsync(
     presentationDefinition: PresentationDefinitionV2,
-    selectedCredentials: OriginalVerifiableCredential[],
+    selectedCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed